internal/consensus/reactor.go

```diff
@@ -1403,3 +1403,7 @@ func (r *Reactor) peerStatsRoutine() {
 		}
 	}
 }
+
+func (r *Reactor) GetConsensusState() *State {
```

Comment:
We probably should not be doing this, but the metrics are important. We can think of a cleaner way maybe in the future.

Reply:
If we think some of the metrics relate to the `consensus.state` is not important info during the  fast-sync,
I can remove the dependency with it and only updates the metrics relate to `block` itself to `reactor.Metrics`

Reply:
@alexanderbez @marbar3778 This PR doesn't include the blockchain `v2`, Do you think is it okay we can just update the gauges relate to `block`? meaning `BlockSizeBytes`, `NumTxs`, `TotalTxs`, `BlockIntervalSeconds`, `CommittedHeight` ?
So it doesn't need `consensus.state` to involve it.

Reply:
Yes, that would be ideal!

Reply:
Okay, I will work on it.